### ISSUE TITLE
Embedding by ID

### ISSUE THREAD

**Issue by ukd1** [rainforestapp/rainforest-cli#45]

**Comment by epaulet** [comment]
Proposed syntax:

 ```
#! login-test
# title some login test

login using "{{login}}" and ..
did it work?
```

```
#! some-other test
# title: some other test

- login-test

click on your profile pic and upload {{random.image}}
do you see the image as your profile pic?
```

**Comment by ukd1** [comment]
@chaselee - fyi, this shipped in 1.2 just now :+1:

**Comment by chaselee** [comment]
:+1: :boom:
